docs/book/mlops-stacks/experiment-trackers/mlflow.md

```diff
@@ -80,6 +80,10 @@ server.
 Token for authenticating with the MLflow tracking server.
 * `tracking_insecure_tls` (optional): Set to skip verifying the MLflow tracking server SSL
 certificate.
+* `databricks_host`: The host of the Databricks workspace with the MLflow managed
```

Comment:
Does it make sense to add a short mention of the `databricks` constant in the `tracking_uri` bullet point above?

Reply:
Along the same lines, I think a short mention in the `how to deploy it` section that mentions databricks would probably make this easier to find?